server/service/osquery_utils/queries.go

```diff
@@ -3201,3 +3204,32 @@ func directIngestHostCertificates(
 
 	return ds.UpdateHostCertificates(ctx, host.ID, host.UUID, certs)
 }
+
+func maybeUpdateLastRestartedAt(now time.Time, host *fleet.Host) error {
```

Comment:
Lint fail on this 'cuz we aren't checking the return. We can drop the return type and drop the return nils to resolve that.